boba/cli.py

```diff
@@ -48,14 +52,46 @@ def print_help(err=''):
     ctx.exit()
 
 
+def run_batch_of_universes(folder, universes):
+    batch = []
+    for universe in universes:
+        batch.append(run_universe(folder, universe))
+
+    return batch
+
+
+def run_universe(folder, script):
+    out = None
+    # choose python or R based on file extension of provided universe
+    if(script.split('.')[1] == "py"):
+        out = subprocess.Popen(["python", "-W", "ignore", script], cwd=folder + "/code/", stdout = PIPE, stderr = PIPE)
+    else:
+        out = subprocess.Popen(["Rscript", script], cwd=folder + "/code/", stdout = PIPE, stderr = PIPE)
+    # it's ok to block here because this function will be running as a seperate process
+    output, err = out.communicate()
+    print(err.decode("utf-8"))
+    print(script + "\n" + output.decode("utf-8"))
+    return (script.split('.')[0], out.returncode)
+
+
+def run_commands_in_folder(folder, file_with_commands):
+    cwd = os.getcwd()
+    os.chdir(folder)
+    with open(file_with_commands) as f:
+        for line in f.readlines():
+            os.system(line)
+    os.chdir(cwd)
+
 @click.command()
 @click.argument('num', nargs=1, default=-1)
 @click.option('--all', '-a', 'run_all', is_flag=True,
               help='Execute all universes')
 @click.option('--thru', default=-1, help='Run until this universe number')
+@click.option('--jobs', default=1, help='number of universes that can be running at a time. Set to 0 to make this the number of cores on the computer.')
```

Comment:
Minor fix: use a very short sentence as `help` (we could put more details in the documentation) and capitalize the first letter